test/conformance/tests/sriov_operator.go

```diff
@@ -540,56 +540,15 @@ var _ = Describe("[sriov] operator", func() {
 			})
 		})
 
-		Context("NAD update", func() {
-			// 24713
-			It("NAD is updated when SriovNetwork spec/networkNamespace is changed", func() {
-				ns1 := "test-z1"
-				ns2 := "test-z2"
-				defer namespaces.DeleteAndWait(clients, ns1, 1*time.Minute)
-				defer namespaces.DeleteAndWait(clients, ns2, 1*time.Minute)
-				err := namespaces.Create(ns1, clients)
-				Expect(err).ToNot(HaveOccurred())
-				err = namespaces.Create(ns2, clients)
-				Expect(err).ToNot(HaveOccurred())
-
-				sriovNetworkName := "test-sriovnetwork"
+		Context("PF shutdown", func() {
+			// 29398
+			It("Should be able to create pods successfully if PF is down.Pods are able to communicate with each other on the same node", func() {
```

Comment:
I don't think this test is right

you need to search for an sriov interface that you can shutdown.